### PR TITLE
Added jump links on mobile for About and Health Info

### PR DESCRIPTION
**Pull request by akampen**

### CODE REVIEW

File: src/pages/Health.tsx [hunk 1]
```diff
@@ -8,17 +9,76 @@ import {
...
+          <Text textAlign="center" fontStyle="italic">
+            Jump to:
+          </Text>
+          <Grid
+            width="100%"
+            templateColumns="repeat(2,1fr)"
+            gap={1}
+            textAlign="center"
+          >
+            <Link gridRow={1} href="/health/#pollution" color="#32bfd1">
```
Comment (CarsonHerness) [inline src/pages/Health.tsx:67]: I think you need to add `id`s to the headings in this file yet

File: src/pages/Health.tsx [hunk 2]
```diff
@@ -8,17 +9,76 @@ import {
...
 const Health: React.FC = () => {
   const {t} = useTranslation('health');
+  const [isMobile, setIsMobile] = useState(
+    window.matchMedia('(max-width: 47.9em)')?.matches ?? false
+  );
+  // -------- Detect screen size for conditional formatting --------- //
+  /**
+   * Adjust UI depending on screenwidth. This function is called from event
+   * listeners with a max-width match media query.
+   * @param this - a media query that either matches or doesn't
```
Comment (CarsonHerness) [inline src/pages/Health.tsx:26]: add note for when "`matches`"

File: src/pages/Health.tsx [hunk 3]
```diff
@@ -8,17 +9,76 @@ import {
...
+  }, []);
+  // -----------------  End detect screen size ----------------- //
 
   return (
     <Flex width="full" align="center" direction="column" padding={8}>
       <Heading as="h1">{t('pageHeading')}</Heading>
+      {isMobile && (
+        <Box>
+          <Text textAlign="center" fontStyle="italic">
+            Jump to:
```
Comment (CarsonHerness) [inline src/pages/Health.tsx:59]: Will need to translate

**Review by CarsonHerness — COMMENTED** [review]
Looks mostly good but some `id`s are missing from the Health page

File: src/pages/Health.tsx (hunk 1 above)
Comment (akampen) [reply src/pages/Health.tsx:67]: Yep totally forgot, ty

**Review by CarsonHerness — COMMENTED** [review]
Okay weird issue that I'm seeing: when anything is added after the `health` in the URL like `/` or `/#protection`, then the images don't load--I have no idea what causes this tho

**Review by CarsonHerness — APPROVED** [review]
Looks good!

**Pull request merged**